### PR TITLE
Appiontment's system

### PR DESCRIPTION
**Pull request by Johngtka**
Added new route to the application -> /visit
User can select a patient from the table to open a new appointment or navigate directly and search for a patient

brave_pmdllPEnDW

### CODE REVIEW

**Comment by Johngtka** [comment]
**checklist:**
[] - When visit loads and there is no patient, show autocomplete and text 'please search for a patient to open a new appointment'

File: src/app/patient-table/patient-table.component.html [hunk 1]
```diff
@@ -1,6 +1,10 @@
 <table mat-table [dataSource]="dataSource" class="mat-elevation-z8">
     <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
-    <tr mat-row *matRowDef="let row; columns: displayedColumns"></tr>
+    <tr
+        mat-row
+        *matRowDef="let row; columns: displayedColumns"
+        (click)="sendVisit(row)"
```
Comment (macrog) [inline src/app/patient-table/patient-table.component.html:6]: `sendVisit` not the best name for a function, maybe: `startNewVisit` or `startNewAppointment` or `newAppointment`

File: src/app/patient-table/patient-table.component.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import { Component } from '@angular/core';
 
+import { Router } from '@angular/router';
```
Comment (macrog) [inline src/app/patient-table/patient-table.component.ts:3]: organise imports please

**Review by macrog — CHANGES_REQUESTED** [review]
Just a few little things to fix and it will be ready

File: src/app/patient-table/patient-table.component.html (hunk 1 above)
Comment (Johngtka) [reply src/app/patient-table/patient-table.component.html:6]: done

File: src/app/patient-table/patient-table.component.ts (hunk 2 above)
Comment (Johngtka) [reply src/app/patient-table/patient-table.component.ts]: done

File: src/app/pipes/age.pipe.ts [hunk 3]
```diff
@@ -1,6 +1,8 @@
-import { Pipe, PipeTransform } from '@angular/core';
+import { Pipe } from '@angular/core';
```
Comment (macrog) [inline src/app/pipes/age.pipe.ts:1]: I don't understand why did you changed this?

**Review by macrog — APPROVED** [review]
checklist: 

- [X] new route on the side menu
- [X] clicking the table redirects to visit route with selected patient
- [X] if there is no patient when navigating to /visit message is show to search for one

**Comment by macrog** [comment]
@Johngtka please provide a squash and merge message first!

**Comment by Johngtka** [comment]
Squash and Merge commit proposal:
feat: Visit's system added (#18):

**Pull request merged**